src/robot/result/model.py

```diff
@@ -419,6 +419,44 @@ def doc(self):
         return ''
 
 
+@Body.register
+class Error(model.Error, StatusMixin, DeprecatedAttributesMixin):
```

Comment:
Shouldn't `Error` be registered also to `Branches` and `Iterations`? Otherwise something like
```robotframework
*** Test Case ***
Invalid syntax in iteration
    WHILE    True
        ELSE
    END

Invalid syntax in branch
    IF    True
        EXCEPT
    END
```
probably won't be handled properly. These cases most likely require new tests.